ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -674,6 +674,22 @@ public void shouldDeleteTopic() {
     assertThat("Expected topic X to be deleted", !topicExists("X"));
   }
 
+  @Test
+  public void shouldCreateStreamWithVariableSubstitution() {
+    // Given:
+    // When:
+    makeKsqlRequestWithVariables(
+        "CREATE STREAM X AS SELECT * FROM " + PAGE_VIEW_STREAM + " WHERE USERID='${id}';",
```

Comment:
This test does not actually validate that variable substitution works. The test would succeed just as well if variable substitution were not supported and `'${id}'` were simply treated as a string key.